package on2015_10.on2015_10_12_Codeforces_Round__325__Div__1_.F___Digits_of_Number_Pi;

import net.egork.chelper.tester.NewTester;

import org.junit.Assert;
import org.junit.Test;

public class Main {
	@Test
	public void test() throws Exception {
		if (!NewTester.test("lib/test/on2015_10/on2015_10_12_Codeforces_Round__325__Div__1_/F___Digits_of_Number_Pi/F - Digits of Number Pi.task"))
			Assert.fail();
	}
}
